include/SDL3/SDL_events.h

```diff
@@ -185,6 +185,11 @@ typedef enum SDL_EventType
     SDL_EVENT_FINGER_UP,
     SDL_EVENT_FINGER_MOTION,
 
+    /* Pinch events */
```

Comment:
Zoom events? It wasn't intuitive for me that a pinch event would also mean the opposite of pinch. :)

Reply:
Although looking through the implementations, at least half of them refer to this as pinch, so this is probably fine. The event is well documented.

Reply:
hum not sure I got it, but yes pinch and zoom are the same thing.
and this can be "pinch in" or "pinch out" for "zoom in/out"

Reply:
I have documented more the events enum started/updated/end